swagger.yaml

```diff
@@ -23,11 +23,70 @@ paths:
               schema:
                 type: string
 
+  /v1/users:
+    get:
+      operationId: listUsers
+      summary: Get list of Severa users.
+      description: Get list of Severa users.
+      tags:
+        - Users
+      parameters:
+        - name: isActive
+          in: query
+          description: Filter by users active state
+          schema:
+            type: boolean
+      responses:
+        "200":
+          description: Get list of Severa users
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/User"
+        "default":
+          description: Invalid request was sent to the server.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+
+  /v1/users/{guid}:
+    get:
+      operationId: findUser
+      summary: Find user by id
+      description: Finds user by id in Severa.
```

Comment:
Aren't you supposed to find users by their Keycloak ID?

Reply:
Resolved